migrator.sh

```diff
@@ -556,13 +556,13 @@ query_source_images() {
       IMAGE_TAGS=""
 
       # set page URL to start with
-      PAGE_URL="https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page=1&page_size=250"
+      PAGE_URL="https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page=1&page_size=${NUM_TAGS}"
 
       # retrieve a list of tags at the target repository
       TAGS_AT_TARGET=$(query_tags_to_skip ${NAMESPACE}/${i})
 
       # loop through each page of tags
-      while [ "${PAGE_URL}" != "null" ]
+      while [ "${PAGE_URL}" != "null" ] && [ "$(count_list ${IMAGE_TAGS})" -le "${NUM_TAGS}" ]
```

Comment:
What is the goal of limiting the number of tags that are returned?  Are you attempting to limit the number of tags that are getting pulled form Docker Hub?